Week1/homework/app.js

```diff
@@ -1,11 +1,150 @@
 'use strict';
 
 {
+  // 1.1 declaring an array 
   const bookTitles = [
     // Replace with your own book titles
     'harry_potter_chamber_secrets',
+    "a_history_of_western_philosophy",
+    'sophie’s_world',
+    'meditations',
+    'zen_and_the_art_of_motorcycle_maintenance',
+    'man’s_search_for_meaning',
+    'the_essential-epicurus' ,
+    'meditations_on_first_philosophy',
+    'critique_of_pure_peason',
+    'beyond_good_and_evil'
   ];
-
+  //1.3 Make a function
   // Replace with your own code
-  console.log(bookTitles);
+  function myFavouriteBooks(books , ul){
```

Comment:
Good that you create a separate function for this. The name is not unclear but I would have called it something like  `displayBooks`.